packages/app/app/page.tsx

```diff
@@ -1,5 +1,11 @@
 import { Stackbox } from "@/components/stackbox/Stackbox";
 
+export const metadata = {
+  title: "Stackly | Stack crypto over time.",
+  description:
+    "Stackly is a simple, non-custodial tool that uses the CoW protocol to place recurring swaps based on DCA..",
+};
```

Comment:
this should be on route layout to be shared across all the app (overridden when we want).
Being on the app page (aka home page) will not be available on other pages we might have.

Reply:
We have got some conflicts with "use client". Don't know how to solve this in the best way.